make/modules/java.base/Lib.gmk

```diff
@@ -233,3 +233,25 @@ ifeq ($(ENABLE_FALLBACK_LINKER), true)
 
   TARGETS += $(BUILD_LIBFALLBACKLINKER)
 endif
+
+################################################################################
+
+ifeq ($(call isTargetOs, linux)+$(call isTargetCpu, x86_64)+$(INCLUDE_COMPILER2), true+true+true)
+    ifeq ($(TOOLCHAIN_TYPE), gcc)
+        $(eval $(call SetupJdkLibrary, BUILD_LIB_X86_64, \
+            NAME := x86_64, \
+            TOOLCHAIN := TOOLCHAIN_LINK_CXX, \
+            OPTIMIZATION := HIGH, \
+            CFLAGS := $(CFLAGS_JDKLIB), \
+            CXXFLAGS := $(CXXFLAGS_JDKLIB), \
+            LDFLAGS := $(LDFLAGS_JDKLIB) \
+                $(call SET_SHARED_LIBRARY_ORIGIN), \
+            LIBS := $(LIBCXX), \
+            LIBS_linux := -lc -lm -ldl, \
+        ))
+
+        TARGETS += $(BUILD_LIB_X86_64)
+  endif
+endif
```

Comment:
Indentation looks off here (https://openjdk.org/groups/build/doc/code-conventions.html)
```suggestion
ifeq ($(call isTargetOs, linux)+$(call isTargetCpu, x86_64)+$(INCLUDE_COMPILER2), true+true+true)
  ifeq ($(TOOLCHAIN_TYPE), gcc)
    $(eval $(call SetupJdkLibrary, BUILD_LIB_X86_64, \
        NAME := x86_64, \
        TOOLCHAIN := TOOLCHAIN_LINK_CXX, \
        OPTIMIZATION := HIGH, \
        CFLAGS := $(CFLAGS_JDKLIB), \
        CXXFLAGS := $(CXXFLAGS_JDKLIB), \
        LDFLAGS := $(LDFLAGS_JDKLIB) \
            $(call SET_SHARED_LIBRARY_ORIGIN), \
        LIBS := $(LIBCXX), \
        LIBS_linux := -lc -lm -ldl, \
    ))

    TARGETS += $(BUILD_LIB_X86_64)
  endif
endif
```

I'm also still wondering about the library name. It's very generic for something that seems to be rather specific.